server/methods/accounts/accounts.app-test.js

```diff
@@ -528,8 +516,7 @@ describe("Account Meteor method ", function () {
           groupId: Random.id(),
           email: fakeUser.emails[0].address,
           name: fakeUser.profile.addressBook[0].fullName
-        })).to.not.throw(Meteor.Error, /Access denied/);
-      return done();
+        })).to.not.throw(Meteor.Error);
```

Comment:
when testing for errors not thrown, it is a good idea to remove the limitation on _type_ of error.